invokeai/backend/model_management/model_manager.py

```diff
@@ -135,6 +169,33 @@ def get_model(self, model_name: str=None)->dict:
             "hash": hash,
         }
 
+    def get_sub_model(
```

Comment:
I think I'd almost prefer a separate method for each of these. Would make the usage a lot more clear, and avoid needing to manually set type hints on the user's end.

Reply:
Ok, sure. Will do.

Reply:
This is now done.